jwst/extract_1d/ifu.py

```diff
@@ -92,6 +92,19 @@ def ifu_extract1d(input_model, ref_dict, source_type, subtract_background,
 
     extract_params = get_extract_parameters(ref_dict, bkg_sigma_clip, slitname)
 
+    # If the user supplied extraction center coords,
+    # load them into extract_params for use later.
+    if center_xy is not None:
+        if len(center_xy) == 2:
+            extract_params['x_center'] = int(center_xy[0])
+            extract_params['y_center'] = int(center_xy[1])
+            log.info(f'Using user-supplied x_center={center_xy[0]}, y_center={center_xy[1]}')
+        else:
+            log.warning('Incorrect number of values in center_xy; should be two.')
```

Comment:
Might still need to add extract_params['x_center'] = None in this else: block, or they will be undefined for the following check on None on L428.

Reply:
Good catch. Fixed in next commit. I initialize them to None and then set them only if given by the user.